Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.h

```diff
@@ -16,10 +16,22 @@ struct JPEGLoadingContext;
 
 // For the specification, see: https://www.w3.org/Graphics/JPEG/itu-t81.pdf
 
+struct JPEGDecoderOptions {
+    enum class CMYK {
+        // 0, 0, 0, 0 is white, 1, 1, 1, 1 is black.
```

Comment:
What are these comments supposed to represent? The raw CMYK values decoded from the JPEG?
If this is true, comments are inverted. We used to call `invert_colors_for_adobe_images` because they are stored in the other way around inside Photoshop's images, and it became the de facto standard.

These comments are just confusing to me, maybe we should just have
```
Normal,
PDF, // CMYK images embeded in PDF don't need Photoshop's special treatment
```

I guess that renaming `Normal` to `Adobe` or `Photoshop` would also work.

Reply:
But PDFs are Adobe too!

I tweaked the comment a bit.